service_upstart_linux.go

```diff
@@ -224,11 +224,11 @@ func (s *upstart) Status() (Status, error) {
 }
 
 func (s *upstart) Start() error {
-	return run("initctl", "start", s.Name)
+	return run("/sbin/initctl", "start", s.Name)
```

Comment:
do we need to do this for [systemctl](https://github.com/evergreen-ci/service/blob/a20f329f17748832fdbff8584b1accf28f322963/service_systemd_linux.go#L186) too? Is it just an upstart thing?

Reply:
We could, but it's less of a problem. `initctl` is only meant to be used by the sudo user since all upstart services are privileged. In comparison, `systemctl` is in `/bin`, likely because regular, unprivileged users are also allowed to create their own systemd services. While `/sbin` is not always in the `PATH` for unprivileged users, `/bin` (which is the typical location for `systemctl`) can be assumed to be in the `PATH` since all users are allowed to use executables in that directory.